config.json

```diff
@@ -432,6 +433,12 @@
       "difficulty": 1,
       "topics": [
       ]
+    },
+    {
+      "slug": "pangram",
+      "difficulty": 1,
+      "topics": [
+      ]
```

Comment:
I don't think it would hurt to add topics now - worst thing that could happen is they change later on, but that wouldn't be all that bad!